internal/models/flow_state.go

```diff
@@ -152,5 +153,17 @@ func (f *FlowState) VerifyPKCE(codeVerifier string) error {
 }
 
 func (f *FlowState) IsExpired(expiryDuration time.Duration) bool {
+	if f.AuthenticationMethod == MagicLink.String() {
```

Comment:
Can expand to more flows, primarily included magic link because it's the most used link based flow.

Would generally expect `Recovery` and `Signup` to be actioned on quickly